app/main.ipynb

```diff
@@ -268,18 +351,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 22,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "result = A[:2]\n",
+    "print(result)"
```

Comment:
The selection of the first two rows using slicing is correct and matches the task requirement.